crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -353,6 +359,9 @@ impl LedgerDB {
 
         self.db.write_schemas(schema_batch)?;
 
+        // Error can be ignored becaue it only errors when there are no receivers
+        let _ = self.soft_confirmation_tx.send(batch_to_store.l2_height);
```

Comment:
Why do this here and not where we call `commit_soft_batch` (sequencer)? Separation of concerns here is violated because we're broadcasting in the storage layer which is not part of it's job. Also this added some extra parameters to initializing the storage layer which are not part of storage requirements.

Reply:
Yup, correct